jsk_perception/node_scripts/image_publisher.py

```diff
@@ -84,30 +84,39 @@ def publish(self, event):
                 info.R = [1, 0, 0, 0, 1, 0, 0, 0, 1]
             self.pub_info.publish(info)
 
-    def cv2_to_imgmsg(self, img_bgr, encoding):
+    def cv2_to_imgmsg(self, img, encoding):
         bridge = cv_bridge.CvBridge()
         # resolve encoding
-        if getCvType(encoding) == 0:
+        if getCvType(encoding) in [0, 2, 5]:
```

Comment:
how about `getCvType(encoding) in [cv2.CV_8UC1, cv2.CV_16UC1, cv2.CV_32FC1]`

Reply:
Thank you, I updated.